sqlmesh/core/renderer.py

```diff
@@ -487,6 +491,16 @@ def render(
                 )
                 raise
 
+            if self._lint_query:
```

Comment:
We don't need to do this every time we render the query. We only need to do this once at load time, no?